### PR TITLE
Travis configured

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary
TravisCI configured, rspec installed and badge in green

## Trello
https://trello.com/c/TZU2ooPi/13-7-configurar-travisci

### CODE REVIEW

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Your PR add Devise, which is consider in the PR #4, change the branch destination from `master` to the correct branch.

File: config/database.travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+test: &test
```
Comment (nicozar95) [inline config/database.travis.yml:1]: I don't think this file is necesary :thinking:

File: config/database.yml [hunk 2]
```diff
@@ -60,7 +60,9 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: kick_test
+  database: wbooks_test
```
Comment (nicozar95) [inline config/database.yml]: Why did you change youur DB configuracion on this Travis PR?

**Review by nicozar95 — CHANGES_REQUESTED** [review]
image

Please change

File: config/database.yml (hunk 2 above)
Comment (alanlamaswolox) [reply config/database.yml]: I did it to match the database's name to the project's name

File: config/database.yml [hunk 3]
```diff
@@ -61,6 +61,8 @@ development:
 test:
   <<: *default
   database: kick_test
+  username: user-wbooks-test
+  password: user-wbooks-test
```
Comment (fedeest) [inline config/database.yml]: Wait, didn't you add this in a previous PR?

And the database name is different. In the previous PR you changed it to `wbooks`

**Review by fedeest — APPROVED** [review]

**Review by nicozar95 — APPROVED** [review]
Great work!!! 
image
